packages/mds-audit/src/api.ts

```diff
@@ -71,16 +70,6 @@ import {
 
 // const AUDIT_PROVIDER_ID = 'dc3dfcf1-ed9f-4606-9c3b-ef19027846ec'
 
-/**
- * convert provider_id to provider name (if available)
- * @param  {UUID} provider_id
- * @return {String} name or provider_id
- */
-// TODO lib
-function providerName(provider_id: UUID): string {
-  return providers[provider_id] ? providers[provider_id].provider_name : provider_id.split('-')[0]
```

Comment:
Notice that this implementation is slightly different than the others. Perhaps this should be the chosen implementation,

Reply:
Yeah, the first chunk of the UUID is certainly 'unique enough' for our purposes.